cou/apps/channel_based.py

```diff
@@ -28,17 +29,17 @@
 class ChannelBasedApplication(OpenStackApplication):
     """Application for charms that are channel based."""
 
-    def _get_latest_os_version(self, unit: COUUnit) -> OpenStackRelease:
-        """Get the latest compatible OpenStack release based on the channel.
+    @property
```

Comment:
Why we need this here? It's not same version as we have in base.py?

Reply:
> It's not same version as we have in base.py?
No. Channel base apps doesn't have a workload version. We decide the OpenStackRelease by what the charm channel says

Reply:
I see, `os_versions[unit.os_release].append(unit.name)` and `os_versions[self.channel_codename].append(unit.name)`.